README.md

```diff
@@ -384,3 +385,47 @@ data:
 ```
 
 When starting kube-httpcache, remember to set the `--backend-watch=false` flag to disable watching the (non-existent) backend endpoints.
+
+## Helm Chart installation
+
+Using [HELM](chart/) to rollout an instance of kube-httpcache.
+
+Ensure your defined frontend and backend services have a port
+name `http`:
```

Comment:
This documentation is inconsistent with the actual chart; here, the frontend service is created as part of the chart, and it has an `http` port no matter what.

Reply:
yes, looks like a misunderstanding to be ensure there is a backend service for Varnish available.